wormhole-connect/src/views/v2/Bridge/Routes/index.tsx

```diff
@@ -108,12 +108,17 @@ const Routes = ({ sortedSupportedRoutes, ...props }: Props) => {
         const isSelected = routeConfig.name === props.selectedRoute;
         const quoteResult = quotesMap[name];
         const quote = quoteResult?.success ? quoteResult : undefined;
+        // Default message added as precaution, as 'Error' type cannot be trusted
+        const quoteError =
+          quoteResult?.success === false
+            ? quoteResult?.error?.message ?? 'Error while getting a quote.'
```

Comment:
Can we add the route name to the error message, like: `Error while getting the quote for ${name}`?